java/bin/start.sh

```diff
@@ -64,10 +64,10 @@ if [ -z "$SKIP_BUILD" ]; then
 
 fi
 
-OPT_SPARK_VERSION="--version=1.1"
+OPT_SPARK_VERSION="--version=1.2"
 OPT_PROJECT="--project=${GCP_PROJECT}"
 OPT_REGION="--region=${REGION}"
-OPT_JARS="--jars=file:///usr/lib/spark/external/spark-avro.jar,${GCS_STAGING_LOCATION}/${JAR_FILE}"
```

Comment:
This will be a breaking change for UI, @surjits254 can you please verify ?
@rajc242  Is this really needed or it is more like a clean up effort ?

Reply:
Spark runtime 1.2 is failing with message that file not found exception.

Reply:
Oh ok. What happens if we remove reference of this jar in both places i.e. start.sh and jenkins file ?
Does it run with both v1.1 and v1.2 ?

Reply:
We didn't change anything for `1.1` so it is working as expected but `1.2` is not working. I explicitly provided jar path through `JARS` option and it is working fine.